frontend/css/src/styles.css

```
@@ -206,6 +206,20 @@
         box-shadow: 0 0 30px rgba(244, 63, 94, 0.08);
     }
 
+    .success-box {
+        @apply mt-2 bg-jade-500/10 border border-jade-500/25 text-jade-300 px-5 py-4 rounded-xl;
+        box-shadow: 0 0 30px rgba(52, 211, 153, 0.1);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)